lib/usb_connection.js

```diff
@@ -14,7 +14,8 @@ var debug = require('debug')('discovery:usb');
 var DFU = require('./dfu');
 var logs = require('./logs');
 
-
+const outVendorcontrolTransferType = 0x40;
+const requestResetUSBD = 0x02;
```

Comment:
What do you think about sticking to writing `const` identifiers in the "traditional" way, eg `ALL_CAPS_WITH_UNDERSCORES`? We could probably set up a jscs rule for it